library/node

```diff
@@ -95,31 +95,31 @@ Directory: 6/wheezy
 
 Tags: 4.8.6, 4.8, 4, argon
 Architectures: amd64, ppc64le, arm64v8, arm32v7, i386
-GitCommit: bf84a38aeacb4f6aad34e07c79fd3a0084da5cd2
+GitCommit: f941101d4d706cea27fab3b628c19e870a78d194
```

Comment:
I'm not sure if we need this change for v4?

Reply:
that was the output of the script

Reply:
Yeah I can verify that on my computer :)